hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -22,12 +22,12 @@
     }
   ],
   wakeup_list: [
-    { name: "aon_gsc_wk",
+    { name: "aon_sysrst_ctrl_wkup_req",
```

Comment:
These names are better, thanks!

Reply:
one more note, i'm guilty of this a lot as well...but if it's already from `sysrst_ctrl`, do we also need to add `sysrst_ctrl` in the name..?

Reply:
We actually don't. I tend to prefer descriptive names for some things so what I said earlier was a knee-jerk reaction. Please ignore it :)

Reply:
Oh so I did this because the generated software constants for the power manager do not have `sysrst_ctrl` in their names, which could potentially lead to conflicts.

Another way would be to change the wakeup name generation for the SW constants.

Reply:
no worries. This is not a big deal. I wouldn't change it just based on my comments.
But post our milestones, i think it'll be worth huddling and figuring out how we want to name things.

We are not very consistent right now (some ip's will add the unique prefix, some won't).  Whatever we decide, we should make sure to update the rest of our tooling flow to match.

Reply:
alright, sounds good!